src/nvim/buffer_defs.h

```diff
@@ -453,6 +453,12 @@ typedef struct {
 /// Primary exists so that literals of relevant type can be made.
 typedef TV_DICTITEM_STRUCT(sizeof("changedtick")) ChangedtickDictItem;
 
+typedef struct {
+  LuaRef on_lines;
+  LuaRef on_changedtick;
+} UpdateCallbacks;
+#define UPDATE_CALLBACKS_INIT { LUA_NOREF, LUA_NOREF }
```

Comment:
would this same `UpdateCallbacks` struct also be used for other types of (future) "attach" mechanisms? If not, maybe it should be named `BufUpdateCallbacks` and `BUFUPDATE_CALLBACKS_INIT`. It's harder to build a mental model if generic names are used for specific things.

Reply:
makes sense, fixed.